website/react-magma-docs/src/pages/api/native-select.mdx

```diff
@@ -0,0 +1,187 @@
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <NativeSelect
+      labelText="Select this"
+      hasError
+      errorMessage="This is an error"
+    >
+      <option>Red</option>
+      <option>Green</option>
+      <option>Blue</option>
+    </NativeSelect>
+  );
+}
+```
+
+## Helper Message
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <NativeSelect
+      labelText="Select this"
+      helperMessage="Helper message appears here"
+    >
+      <option>Red</option>
+      <option>Green</option>
+      <option>Blue</option>
+    </NativeSelect>
+  );
+}
+```
+
+## Inverse
+
+The `isInverse` prop will render a dark background and light text. The children will
+inherit the `isInverse` prop unless specified otherwise.
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse style={{ marginBottom: magma.spaceScale.spacing04 }}>
+      <CardBody>
+        <NativeSelect labelText="Select this" isInverse>
+          <option>Red</option>
+          <option>Green</option>
+          <option>Blue</option>
+        </NativeSelect>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
+## Inverse disabled
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse style={{ marginBottom: magma.spaceScale.spacing04 }}>
+      <CardBody>
+        <NativeSelect labelText="Select this" isInverse disabled>
+          <option>Red</option>
+          <option>Green</option>
+          <option>Blue</option>
+        </NativeSelect>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
+## Inverse with error
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse style={{ marginBottom: magma.spaceScale.spacing04 }}>
+      <CardBody>
+        <NativeSelect
+          labelText="Select this"
+          isInverse
+          hasError
+          errorMessage="This is an error"
+        >
+          <option>Red</option>
+          <option>Green</option>
+          <option>Blue</option>
+        </NativeSelect>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
+## Inverse Helper Message
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse style={{ marginBottom: magma.spaceScale.spacing04 }}>
+      <CardBody>
+        <NativeSelect
+          labelText="Select this"
+          isInverse
+          helperMessage="Helper message appears here"
+        >
+          <option>Red</option>
+          <option>Green</option>
+          <option>Blue</option>
+        </NativeSelect>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
```

Comment:
We might also want to show an example with an onChange event, to demonstrate getting the select's value.